services/motion/builtin/move_request.go

```diff
@@ -655,6 +665,9 @@ func (ms *builtIn) relativeMoveRequestFromAbsolute(
 	startPoseInv := spatialmath.PoseInverse(startPose.Pose())
 
 	goal := referenceframe.NewPoseInFrame(referenceframe.World, spatialmath.PoseBetween(startPose.Pose(), goalPoseInWorld))
+	if spatialmath.PoseAlmostCoincidentEps(goal.Pose(), spatialmath.NewZeroPose(), motionCfg.planDeviationMM) {
```

Comment:
This will still have weird behavior if the orientation is off.

If we issue a command in position-only mode to our current location, but 180 degree different orientation, then we should do nothing and declare victory.

If we issue a command in *NOT* position-only mode to our current location, but 180 degree different orientation, then we should plan to that pose. Such a plan may be non-trivial, especially if it is a robot that cannot turn in place.